client/src/components/OverlayComponent/OverlayComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './OverlayComponent.scss';
+import ActionButton, {ActionButtonColor} from 'components/ActionButton/ActionButton.tsx';
+
+interface AddNotificationChannelModalProps {
+    onClose: () => void;
+}
+
+const OverlayComponent: React.FC<AddNotificationChannelModalProps> = ({ onClose }) => {
+    return (
+        <div className="modal-overlay">
+            <div className="modal-content">
+                <p>List placeholder</p>
+                <ActionButton
+                    onClick={onClose}
+                    description="Close"
+                    color={ActionButtonColor.RED}
+                />
+            </div>
+        </div>
+    );
+};
```

Comment:
Why does this component has AddNotifiationChannelModalProps while it seems to be a pretty generic component?

Reply:
Also what's the purpose of this component if it doesn't have any meaningful props?